# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/nvi/no-catman.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


This catman stuff is stuipid - I don't like it!

 - Clifford

--- ./Makefile~	Mon Oct 26 15:42:21 1998
+++ ./Makefile	Mon Oct 26 15:49:38 1998
@@ -30,7 +30,7 @@
 	install -o root -g bin -m 555 bin/nvi $(BINDIR)/nvi
 	(cd $(BINDIR); ln -sf nvi nex)
 	install -o root -g root -m 444 USD.doc/vi.man/vi.1 $(MANDIR)/$(MANPREFIX)vi.1
-	install -o root -g root -m 444 USD.doc/vi.man/vi.0 $(CATMANDIR)/$(MANPREFIX)vi.1
+#	install -o root -g root -m 444 USD.doc/vi.man/vi.0 $(CATMANDIR)/$(MANPREFIX)vi.1
 
 clean: PORT/linux/Makefile
 	make -C PORT/linux $(MFLAGS) clean
